add_menufunc.js

```diff
@@ -0,0 +1,50 @@
+const { body } = document;
+const section = document.createElement('section');
+const i0 = document.createElement('i');
+const ul = document.createElement('ul');
+const li1 = document.createElement('li');
+const li2 = document.createElement('li');
+const li3 = document.createElement('li');
+const a1 = document.createElement('a');
+const a2 = document.createElement('a');
+const a3 = document.createElement('a');
+
+function MobileMenu() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
This issue is well resolved! ✔️